mappings/net/minecraft/world/biome/BiomeGenerator.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS none/ajk net/minecraft/world/biome/BiomeGenerator
-	FIELD a isDecorating Z
+	FIELD a decorating Z
```

Comment:
I don't really like this all that much myself, I think it should be prefixed with 'is' or 'can' when there's no getters or setters and it's a public field. If there are getters and setters then I think the guidelines make sense.

Thoughts?